mobile-widgets/qml/DiveDetailsEdit.qml

```diff
@@ -120,6 +120,7 @@ Item {
 
 		Qt.inputMethod.hide()
 		// now make sure we directly show the saved dive (this may be a new dive, or it may have moved)
+		manager.selectDive(dive_id)
```

Comment:
Do you have a test case where this is necessary? The undo command should select the affected dive.

Reply:
I added debug output when I was first trying to track this down and thought that I saw an unselected dive and that's when I added that line (which by itself wasn't enough to fix the problem, of course). I have now tried to remove it and can't reproduce a problem without it, so... I guess it's not needed after all. ?!?
Dunno.

Reply:
If it is not needed - please don't add it. Sooner or later it will cost us time trying to figure out whether it can be removed.

Reply:
ok